README.md

```diff
@@ -286,15 +286,50 @@ curl --header "Content-Type: application/json" \
 You should receive a json response matching something like the following:
 ```
 {
-	"caseId": {CASE_ID},
-	"status": "PASS",
-	"message": ...
+    "caseId": {CASE_ID},
+    "status": "PASS",
+    "message": ...
 }
 ```
 
+### Haskell
+
+Haskell testing uses [HUnit](https://hackage.haskell.org/package/HUnit) to run tests.
+
+The same procedure followed for [Python 3](#sample-walkthrough) can be followed here with some notable differences
+
+The differences are:
+
+1. The Suite file to be passed must import the main program file.

```

Comment:
Not sure what you mean by 'passed' here...

Reply:
I mean the uploaded test suite file.

Something like this.
```suggestion
1. The Suite file to be uploaded must import the main program file.
```

Reply:
I see. Isin't it obvious that you need to import the thing you want to test?

Reply:
I thought you absolutely don't trust the user...? 😆 

I would keep it just to make it explicite.

Reply:
I mean their test is gonna flop otherwise 🤣, sure we can leave it in.